homeassistant/components/alarm_control_panel/egardia.py

```diff
@@ -128,6 +57,15 @@ def __init__(self, name, egardiasystem, rs_enabled=False, rs_codes=None):
             self._rs_codes = rs_codes[0]
```

Comment:
You have to look the whole server code logic over, with the new config schema. This will raise a keyerror, since `0` is not in the dictionary. Please also look at `lookupstatusfromcode`. It doesn't look like it's doing the right thing.